test/old_tests/UnitTests/array.cpp

```diff
@@ -909,6 +911,31 @@ TEST_CASE("array_view,array")
     }
 }
 
+// Tests array_view construction for various cv-qualified conversions
+TEST_CASE("array_view,cv array_view")
```

Comment:
Can we add a com_array to array_view test since that was the original issue, just to be safe?